base/collections.jl

```diff
@@ -82,6 +90,11 @@ end
 
 
 # Turn an arbitrary array into a binary min-heap in linear time.
+"""
+    heapify!(v, [ord])
+
+In-place [``heapify``](:func:`heapify`).
```

Comment:
Should be single backticks for code. Double backticks now produce latex. Also in the `PriorityQueue` and `sprand` below.